src/Plugin/Field/FieldFormatter/StrawberryPannellumFormatter.php

```diff
@@ -206,6 +176,7 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
          }
       }*/
       $i = 0;
+      $iiifhelper = new IiifHelper($this->getIiifUrls()['public'], $this->getIiifUrls()['internal']);
```

Comment:
Just because i'm picky. It would be even more optimal if it goes after the if $json[key]. No need to create the object if we don't have the key